src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/Utils.java

```diff
@@ -34,6 +49,44 @@ public static void clearPreferences(Context context) {
         }
     }
 
+    public static void hideKeyboard(Context context, View view) {
+        InputMethodManager inputMethodManager = (InputMethodManager)context.getSystemService(Activity.INPUT_METHOD_SERVICE);
+        inputMethodManager.hideSoftInputFromWindow(view.getWindowToken(), 0);
+    }
+
+    public static void showErrorDialog(final FragmentActivity activity, String error) {
+        DialogFragment fragment = ErrorDialogFragment.newInstance(R.string.validation_errors, error);
+        fragment.show(activity.getSupportFragmentManager(), "error");
+    }
+
+    public static boolean isEmailValid(String email) {
+        String expression = "^[\\w\\.-]+@([\\w\\-]+\\.)+[A-Z]{2,4}$";
+        Pattern pattern = Pattern.compile(expression, Pattern.CASE_INSENSITIVE);
+        Matcher matcher = pattern.matcher(email);
+        return matcher.matches();
+    }
+
+    public static void showAlertDialog(Activity activity, String title, String msg) {
+        Log.d(TAG, "Showing alert dialog...");
+        if (Looper.myLooper() == null) {
+            Looper.prepare();
+        }
+
+        AlertDialog alertDialog = new AlertDialog.Builder(activity).create();
+        alertDialog.setTitle("Lantern");
```

Comment:
Seems like you technically want this to be `title` here @atavism, or remove the title parameter.